solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -114,4 +114,64 @@ print(ans)
 ```
 
 </PySection>
+
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class GCDBoard {
```

Comment:
wait can you use Kattio rather than normal IO

Reply:
not needed